spec/features/guide_publishing_flow_spec.rb

```diff
@@ -161,17 +161,16 @@
     end
   end
 
-  it "should save a draft locally, sent it to Publishing API, then redirect to the front end when previewing" do
+  it "should save a draft locally, send it to Publishing API, then redirect to the front end when previewing" do
```

Comment:
This doesn't quite reflect what it's up to anymore as we're no longer redirecting.